nailgun/entities.py

```diff
@@ -7580,6 +7580,73 @@ def read(self, entity=None, attrs=None, ignore=None, params=None):
         return super(System, self).read(entity, attrs, ignore, params)
 
 
+class TailoringFile(
+        Entity,
+        EntityCreateMixin,
+        EntityDeleteMixin,
+        EntityReadMixin,
+        EntitySearchMixin,
+        EntityUpdateMixin):
+    """A representation of a Tailoring File entity."""
+
+    def __init__(self, server_config=None, **kwargs):
+        self._fields = {
+            'name': entity_fields.StringField(
+                required=True,
+                str_type='alpha',
+                length=(4, 30),
```

Comment:
Is it mentioned somewhere about the length limit? I don't think so.

What If I provide name of 3 chars , will that fail ?